__tests__/index.spec.js

```diff
@@ -109,5 +109,18 @@ describe('onDuplicateUpdate', () => {
         email: 'updated-email',
       }));
     });
+
+    it('should correctly escape field with ? character', async () => {
+      await db
+        .insert({ id: 7, name: '?' })
```

Comment:
put here a different value for `name` (you want to assure that the second insert will update it)